src/Utility/Hash.php

```diff
@@ -209,11 +209,11 @@ protected static function _splitConditions(string $token): array
     /**
      * Check a key against a token.
      *
-     * @param string $key The key in the array being searched.
+     * @param mixed $key The key in the array being searched.
```

Comment:
We should avoid mixed by using string|int etc

Reply:
Well you can't since `Hash` class deals with arrays where keys don't always have to be string.

Reply:
True, but usually scalar. As such better to be explicit. Right now you would also allow objects or other arrays etc, thats bad.

Reply:
Specifying a list of all scalar type vs mixed isn't going to provide any practical benefit. Won't make any difference to tools and humans can already know it takes valid array key types by reading description.

Reply:
Why? You then know that it cannot be any random object for example. As such the benefit is 10x more than mixed already by design.